mmv1/products/monitoring/UptimeCheckConfig.yaml

```diff
@@ -324,9 +333,6 @@ properties:
   - !ruby/object:Api::Type::NestedObject
     name: tcpCheck
     description: Contains information needed to make a TCP check.
-    exactly_one_of:
-      - http_check
-      - tcp_check
     properties:
```

Comment:
What is the reason to remove these lines?

Reply:
I updated the PR description to include information about this (second bullet). The gist is that the new api field does not require either of these (and we dont serve them back / use them if they are provided), so removing the exactly_one_of was the only way around this. If there was a "one_of" field like on protos which acts as a "at most one of", then I would use that.

Also this was the cause of my acceptance test failure rabithole -- https://yaqs.corp.google.com/eng/q/6285662784902397952